labs/multiverse/src/main.rs

```diff
@@ -315,37 +321,46 @@ impl App {
         self.state.global_mode = mode;
     }
 
-    async fn handle_global_key_press(&mut self, key: KeyEvent) -> Result<bool> {
+    async fn handle_global_key_press(&mut self, event: Event) -> Result<bool> {
```

Comment:
The function name is now a lie, since it can call into an event handler for a non-key event; let's maybe rename it `handle_global_event`?

Reply:
Sure: https://github.com/matrix-org/matrix-rust-sdk/pull/4938/commits/a3795e7038520cee4fae614e74f71911a385ddb8.